app/views/hosts/_selected_hosts.html.erb

```diff
@@ -33,8 +36,9 @@
           <% end %>
         </tr>
       <% end %>
-    </tbody>
-  </table>
-  <%= check_box_tag "keep_selected", "", false, :title => _("Keep selected hosts for a future action") %>
-  <%= _('Keep selected hosts for a future action') %><br/><br/>
+      </tbody>
+    </table>
+    <%= check_box_tag "keep_selected", "", false, :title => _("Keep selected hosts for a future action") %>
+    <%= _('Keep selected hosts for a future action') %><br/><br/>
```

Comment:
I've thought at the beginning that it useless due to cookie implementation , but I've managed not to expand this cookie :smile: